railties/lib/rails/test_unit/minitest_plugin.rb

```diff
@@ -86,8 +85,9 @@ def run(args = [])
   def self.plugin_rails_init(options)
     ENV["RAILS_ENV"] = options[:environment] || "test"
 
-    # If run via `ruby` we've been passed the files to run directly.
-    unless run_via[:ruby]
+    # If run via `ruby` we've been passed the files to run directly, or if run
+    # via `rake` then they have already been included.
```

Comment:
`included` is a new term, let's say `eagerly required` to stick with the established terminology.